katas/pokerhands/israel/poker.js

```diff
@@ -0,0 +1,433 @@
+/*Value is the this form:
+xyyzz
+x : Type of hand (Hight card, pair, tri, full.....) Hight card = 1, pair = 2, tri = 3 ......
+yy : max value of type (for example for full house [3d, 3c, 3c, 2d, 2h], this number is "03")
+zz : hight card only if necessary (for example for Four of a kind [kd, kc, kh, kS, 2s], this number is "02" )
+
+so this hand ['KC', '3C', '4C', '6C', '7C'] = 11300 points
+
+*/
+export class pokerGame {
+    getScoreHand(hand) {
```

Comment:
this method is really big, you might need a refactor to separate concerns. If this would kept as is, it can cause maintainability issues in the future

Reply:
yes, i going to try to split the class